### PR TITLE
new footer links

### PR DESCRIPTION
**Pull request by mrtrader1313**
[EXAMPLE]
Status: [BLOCKED][WIP][WAITING FOR REVIEW]

Background (user-story):
1. Bob wants to buy Bitcoins directly to his phone, since he does not have an external wallet.

Implementation:
1. Wallet service using local storage
2. Cold Signing
3. Export key using BIP39 memorable phrases

Unit tests included:
1. Wallet generation
2. Cold signing
3. Key impporting and exporting

Integration tests:
1. Make an order to a local wallet

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -14,14 +14,14 @@ const COMPLIANCE = [
 ]
  const COMPLIANCE2 =[
 
-  {img:'bestchange', url: "https://bestchange.com" , name: 'bestchange'},
-  {img: 'okchanger', url: "https://okchanger.com" , name: 'okchanger'},
+  {img:'bestchange', url: "https://www.bestchange.net/nexchange-exchanger.html" , name: 'bestchange'},
```
Comment (BeOleg) [inline src/components/Footer/index.js]: why bestchange.net? also should be RU for RU customer @chiragneb

Comment (BeOleg) [reply src/components/Footer/index.js]: @mrtrader1313

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -14,14 +14,14 @@ const COMPLIANCE = [
...
+  {img: 'okchanger', url: "https://www.okchanger.com/exchangers/n.exchange" , name: 'okchanger'},
   {img: 'kurs', url: "https://kurs.expert" , name: 'kurs'},
   {img:'exchangesumo', url: 'https://exchangesumo.com/', name: 'exchangesumo'},
   {img:'emon', url: 'http://e-mon.ru/', name: 'emon'},
   {img:'allchange', url: 'https://allchange.org/', name: 'allchange'},
   {img:'bestcurs', url: 'https://bestcurs.org/', name: 'bestcurs'} 
-
+
```
Comment (BeOleg) [inline src/components/Footer/index.js]: This diff is redundant @mrtrader1313

File: src/components/Footer/index.js (hunk 1 above)
Comment (mrtrader1313) [reply src/components/Footer/index.js]: Ok fixing

**Pull request closed without merging**